Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/Migration/ObjectManipulatorMigrationHandler.cs

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Experimental.UI;
+using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Utilities
+{
+    /// <summary>
+    /// Interface defining a migration handler, which is used to migrate assets as they
+    /// upgrade to new versions of MRTK.
+    /// </summary>
+    public class ObjectManipulatorMigrationHandler : IMigrationHandler
```

Comment:
Where / how is this used?

Reply:
this is not used right now - we removed the migration buttons for now because we don't want ppl to upgrade yet - this will be used again once we are ready for graduating the feature.

Reply:
I was using this for testing (I had a button in manipulation handler inspector that migrated to object manipulator). We didn't want to keep this button around in development, but we can use this Migration stuff in future (see #6911).

Reply:
Thanks for the context!